addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -3146,6 +3146,44 @@ options.registry.ConditionalVisibility = options.Class.extend({
 });
 
 options.registry.WebsiteAnimate = options.Class.extend({
+    /**
+     * @override
+     */
+    async start() {
+        await this._super(...arguments);
+        this.isAnimatedText = this.$target.hasClass('o_animated_text');
+        this.$optionsSection = this.$overlay.data('$optionsSection');
+    },
+    /**
+     * @override
+     */
+    onFocus() {
+        if (this.isAnimatedText) {
+            const $toolbar = this.options.wysiwyg.toolbar.$el;
+            $toolbar.append(this.$el);
+            this.$optionsSection.addClass('d-none');
+        }
+    },
+    /**
+     * @override
+     */
+    onBlur() {
+        if (this.isAnimatedText) {
+            this.$optionsSection.append(this.$el);
+        }
+    },
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    async updateUIVisibility() {
+        await this._super(...arguments);
+        this.$el.find('we-button[data-name="no_animation_opt"]').toggleClass('d-none', this.isAnimatedText);
```

Comment:
Why can't this use `_computeWidgetVisibility`? Seems to be a widget with a name, it should work?